frontend/src/pages/ManagerPage.tsx

```diff
@@ -1,16 +1,42 @@
+
 //Manager Dashboard
 import { useState } from "react";
 
 import Header from "../components/Header";
+import YearCardList from "../components/YearCardList";
+
+import '../styles/DeleteButton.css'; // Import the CSS file
+import '../styles/YearCard.css'; // Import the CSS file
+
+import '../styles/YearCardList.css'; // Import the CSS file
+
+import "../styles/InteractionsPane.css";
 import EmployeeList from "../components/EmployeeList";
+
+import "../styles/InteractionsPane.css";
+//import EmployeeList from "../components/EmployeeList";
 import AttachmentList from "../components/AttachmentList";
 import PerformanceReviewButton from "../components/PerformanceReviewButton";
 
 import "../styles/InteractionsPane.css"; //.interactions-section, .elected-employee-container, .previous-years-container
 import "../styles/ManagerSection.css";
 import "../styles/Attachments.css"; //.attachments-container
 
-function ManagerPage() {
+
+
+const ManagerPage: React.FC = () => {

```

Comment:
take the const ManagerPage: React.FC = () => { and change it back to the original function